bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -281,12 +280,13 @@ void PreProcessor::finalizePreProcessing(NodeIdType clientId, SeqNum reqSeqNum)
                                                      HAS_PRE_PROCESSED_FLAG,
                                                      reqSeqNum,
                                                      clientEntry->clientReqInfoPtr->getMyPreProcessedResultLen(),
-                                                     clientEntry->clientReqInfoPtr->getMyPreProcessedResult());
+                                                     clientEntry->clientReqInfoPtr->getMyPreProcessedResult(),
+                                                     cid);
   }
   incomingMsgsStorage_->pushExternalMsg(move(clientRequestMsg));
   preProcessorMetrics_.requestSentForFurtherProcessing.Get().Inc();
   releaseClientPreProcessRequest(clientId, reqSeqNum);
-  LOG_DEBUG(GL, "Pre-processing completed for clientId=" << clientId << " reqSeqNum=" << reqSeqNum);
+  LOG_INFO(GL, "Pre-processing completed for clientId=" << clientId << " reqSeqNum=" << reqSeqNum);
```

Comment:
Please return it back to the debug level.

Reply:
done